### PR TITLE
Add MacvlanNetwork CR example

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ spec:
     }
 ```
 
-Can be found at: `example/deploy/crds/mellanox.com_v1alpha1_macvlannetwork_cr.yaml`
+Can be found at: `./deploy/crds/mellanox.com_v1alpha1_macvlannetwork_cr.yaml`
```
Comment (adrianchiris) [inline README.md:212]: can you copy the file under example folder  (all example files are there)? i think it was just missing.

Comment (e0ne) [reply README.md:212]: It will lead to maintaining two copies of the file. Do we want it?

Comment (adrianchiris) [reply README.md:212]: i would change it to some generic placeholder under /deploy/  similarly to what was done in:

https://github.com/Mellanox/network-operator/blob/master/deploy/crds/mellanox.com_v1alpha1_nicclusterpolicy_cr.yam

Comment (adrianchiris) [reply README.md:212]: the operator-framework creates some "sample" CRs when auto-generating the CRDs, these usually should have no meaningful content. i must have missed that mamduh put this here and not under example folder

Comment (e0ne) [reply README.md:212]: It's already in /deploy directory: https://github.com/Mellanox/network-operator/tree/master/deploy/crds

**Review by adrianchiris — CHANGES_REQUESTED** [review]
as discussed, its better to have the macvlan cr under `example`

and have a simple template under `deploy/crds`

**Review by moshe010 — APPROVED** [review]

**Pull request merged**